controllers/imagerepository_controller.go

```diff
@@ -291,6 +346,46 @@ func (r *ImageRepositoryReconciler) scan(ctx context.Context, imageRepo *imagev1
 		} else {
 			logr.FromContext(ctx).Info("No image credentials secret referenced, and ECR authentication is not enabled. To enable, set the controller flag --aws-autologin-for-ecr")
 		}
+	} else if strings.Contains(imageRepo.Spec.Image, "gcr.io") {
```

Comment:
If someone decides to register `abcdgcr.io` and host images on it, this is going to magically fail for them.